ot_rtos/apps/CMakeLists.txt

```diff
@@ -53,15 +62,27 @@ add_custom_target(
         ${CLOUDIOT_REGION} ${CLOUDIOT_PRIV_KEY}
 )
 
-add_dependencies(test_app client_cfg)
+add_dependencies(otr_test_app client_cfg)
 
-if (${PLATFORM_NAME} STREQUAL nrf52)
-    add_library(demo_101
+if (${OT_PLATFORM} STREQUAL nrf52 AND OTR_DEMO_101)
```

Comment:
quota `${OT_PLATFORM}`. Otherwise, it will result in cmake syntax error if `OT_PLATFORM` is left empty (by default).